riscv/runtime/src/lib.rs

```diff
@@ -45,9 +45,8 @@ extern "Rust" {
 }
 #[no_mangle]
 #[start]
-pub unsafe extern "C" fn __runtime_start() -> ! {
+pub unsafe extern "C" fn __runtime_start() {
     unsafe {
         main();
     }
-    loop {}
```

Comment:
Why is the loop not needed anymore?

Reply:
let's discuss on #392

Reply:
sure, but shouldn't this PR be just the cli file?

Reply:
yes, need to rebase

Reply:
ahhh just noticed that this wasn't targeting `main`, ok will ignore for now then